js/script.js

```diff
@@ -11,83 +11,83 @@
 //When user inputs text into textbox, clicking enter will submit it
 var inputByEnter = document.getElementById("textbox");
 
-inputByEnter.addEventListener("keyup", function(event){
-	//13 signifies the Enter key
-	if(event.keyCode === 13){	
-		//This cancels the default action if necessary
-		event.preventDefault();
-		//Call the addItem() function below
-		addItem();
-	}
+inputByEnter.addEventListener("keyup", function (event) {
+  //13 signifies the Enter key
+  if (event.keyCode === 13) {
+    //This cancels the default action if necessary
+    event.preventDefault();
+    //Call the addItem() function below
+    addItem();
+  }
 });
 
 
 //Create a new item on the list
-function addItem() {
+function addItem () {
 
-	//Create new list element
-	var li = document.createElement("li");
+  //Create new list element
+  var li = document.createElement("li");
 
-	//Set a 'contentEditable' attribute to keep track of whether checkbox is checked
-	li.setAttribute('contentEditable', 'false');
+  //Set a 'contentEditable' attribute to keep track of whether checkbox is checked
+  li.setAttribute('contentEditable', 'false');
 
-	//Create a checkbox
-	var checkbox = document.createElement("INPUT");
-	checkbox.setAttribute("type", "checkbox");
-	// checkbox.setAttribute("onclick", "toggleChecked();") //no longer needed
-	checkbox.className += ' listcheckbox';
-	li.appendChild(checkbox);
+  //Create a checkbox
+  var checkbox = document.createElement("INPUT");
+  checkbox.setAttribute("type", "checkbox");
+  // checkbox.setAttribute("onclick", "toggleChecked();") //no longer needed
+  checkbox.className += ' listcheckbox';
+  li.appendChild(checkbox);
 
-	//Fetch the user input data
-	var inputItem = document.getElementById("textbox").value;
+  //Fetch the user input data
+  var inputItem = document.getElementById("textbox").value;
 
-	//Create a text node with the input data
-	// var newNode = document.createTextNode(inputItem);
-	// li.appendChild(newNode);
+  //Create a text node with the input data
```

Comment:
Good rule of thumb is to delete commented out code if you're not using it. You'll be surprised by how often you wind up never using your commented out code.